doc/en/user/source/webadmin/index.rst

```diff
@@ -11,6 +11,15 @@ The Web administration interface is a web-based tool for configuring all aspects
 
 The following sections detail the menu options available in GeoServer. **Unless otherwise specified, you will need to be logged in with administrative credentials to see these options.**
 
+Choosing the UI language
+------------------------
+
+The administration interface is displayed using the browser native language, if a translation

```

Comment:
```suggestion
The administration interface is displayed using the browser's preferred language when available,
otherwise it will fall back to English.
```